backend/build.rs

```diff
@@ -1,22 +1,48 @@
-use std::{fs::OpenOptions, io::Write, path::PathBuf};
-
 use ethers::prelude::Abigen;
+use std::{fs::OpenOptions, io::Write, path::PathBuf};
 
 fn main() {
```

Comment:
Is everything mentioned in the read me according to this build script still valid after your modficiations?